setup.py

```diff
@@ -1,4 +1,4 @@
-from distutils.core import setup, Extension
+from setuptools import setup, Extension
```

Comment:
This commit would be better as a separate pull request.

In general, it's not a good idea to include multiple, vastly disparate changes in a single PR.